### PR TITLE
In block details, add prev and next buttons 

### PR DESCRIPTION
**Pull request by csillag**
As requested on Slack by @kostko 
Design is [here]

The implementation looks like this:
image

### CODE REVIEW

**Comment by csillag** [comment]
Friendly reminder: this is waiting for input from @donouwens

**Comment by csillag** [comment]
Now this has the blessing of @donouwens , so ready for review!

File: .env [hunk 1]
```diff
@@ -2,7 +2,7 @@ REACT_APP_BUILD_DATETIME=0
 REACT_APP_BUILD_SHA=sha0000000000000000000000000000000000000
 REACT_APP_BUILD_VERSION=
 # REACT_APP_API=http://localhost:8008/v1/
-REACT_APP_API=https://nexus.stg.oasis.io/v1/
-REACT_APP_TESTNET_API=https://testnet.nexus.stg.oasis.io/v1/
-# REACT_APP_API=https://nexus.oasis.io/v1/
-# REACT_APP_TESTNET_API=https://testnet.nexus.oasis.io/v1/
+# REACT_APP_API=https://nexus.stg.oasis.io/v1/
```
Comment (buberdds) [inline .env:5]: revert ?

File: src/app/pages/BlockDetailPage/index.tsx [hunk 2]
```diff
@@ -53,6 +60,47 @@ export type BlockDetailRuntimeBlock = RuntimeBlock & {
   markAsNew?: boolean
 }
 
+const PrevBlockButton: FC<{ scope: SearchScope; currentRound: number }> = ({ scope, currentRound }) => {
```
Comment (buberdds) [inline src/app/pages/BlockDetailPage/index.tsx:63]: optional: If we plan to use it in Consensus it's worth to put this into some common components folder.

File: .env (hunk 1 above)
Comment (csillag) [reply .env:5]: It's in a separate commit, which I intend to drop before merging.

Comment (buberdds) [reply .env:5]: ahh Ok

**Review by buberdds — APPROVED** [review]

File: src/app/components/BlockNavigationButtons/index.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  )
+}
+
+export const NextBlockButton: FC<{ scope: SearchScope; currentRound: number }> = ({
+  scope,
+  currentRound,
+}) => {
+  const { latestBlock } = useRuntimeFreshness(scope)
+  const { t } = useTranslation()
+  const disabled = latestBlock === currentRound
```
Comment (lukaw3d) [inline src/app/components/BlockNavigationButtons/index.tsx:43]: I would have allowed users to go into 404 rather than have an extra request :D

Comment (csillag) [reply src/app/components/BlockNavigationButtons/index.tsx:43]: My understanding is that we are checking the status for every page render anyway, to see if we need to display a banner or something. Results are cached, so checking it again (during the same page render) doesn't actually result in an extra request.

Comment (csillag) [reply src/app/components/BlockNavigationButtons/index.tsx:43]: See https://github.com/oasisprotocol/explorer/pull/1201 for this improvement (the "live" version of the next button)

**Pull request merged**